model/userfollower.go

```diff
@@ -0,0 +1,12 @@
+package model
+
+import (
+	"github.com/google/uuid"
+)
+
+// A Follower is the IRI of someone who follows a user.
+type Follower struct {
+	ID     string `gorm:"primary_key"`
```

Comment:
I want to say you don't need to tag ID with primary_key because magic, but I also dislike magic so... I'm fine with leaving it for now